Framework/src/CheckInterface.cxx

```diff
@@ -59,4 +60,9 @@ void CheckInterface::endOfActivity(const Activity& activity)
   // noop, override it if you want.
 }
 
+shared_ptr<MonitorObject> CheckInterface::retrieveReference(std::string path, int referenceRun, Activity activity)
```

Comment:
here reference run uses `int`, elsewhere `getReferencePlot` uses `uint32_t`. Should it be unified for better error handling (e.g. negative run numbers)?

Reply:
agreed